### DISCUSSION TITLE
Add To-Do Reminder

### DISCUSSION THREAD

**Discussion by drtobbyas** [standardnotes/forum#3280] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
I really love the idea behind standardnotes and it's feature-sets. I want to see a To-Do Reminder on the feature list i.e Ability to set a time to be reminded of a To-Do or notes. This will be really great addition to the app. 

**Describe the solution you'd like**
I use the To-Do a lot and I would like to be able to set reminders for my To-Do similar to what's achievable with Any.do app.

**Describe alternatives you've considered**
I still sometimes have to duplicate some to-dos in Any.do app in order to get a reminder. I want to be all in on Standardnotes because It has almost all the documentation and note-taking features I need

**Comment by 100110010111** [comment]
+1

**Comment by markwylde** [comment]
This would be amazing

**Comment by absurdity9** [comment]
+1

Being able to have reminders in app, and ideally as notifications or emails would supercharge my notes.

**Comment by HaimRSk8r** [comment]
I would also love to be able to set notifications or alarms for specific notes in the app. Currently using another app just for this purpose and prefer to have all of my notes needs covered in Standard Notes.

**Comment by taykuctd** [comment]
Competition already has it covered. SN please step up your game and implement reminders

**Reply by cronkish** [reply]
Second this, please equip SN with comprehensive reminder app functions, like Apple but with color coding,

**Comment by RiaGill** [comment]
Having trouble with the Reminder app not showing up in your notifications settings? Don't worry, it's a fairly common issue. Here are a few things you can try to fix it:-
First, check your notification settings:- Go to Settings > Notifications and make sure the Reminder app is listed. If it's there, make sure the "Allow Notifications" switch is turned on.
Next, try restarting your iPhone:- Sometimes, a simple restart can resolve these kinds of glitches. Just press and hold the side button and one of the volume buttons until the power off slider appears, then swipe to power off. Wait a few seconds, then turn it back on.
Check your app permissions:- Go to Settings > Privacy > Notifications and make sure the Reminder app has permission to send you notifications.

If none of these solutions work, you can try to follow for more instruction below.
https://thegeekpage.com/reminder-app-not-showing-in-notifications-settings-fix/

**Reply by 100110010111** [reply]
This is spam

**Comment by stefhf** [comment]
Has there ever been an official reply to this request? It would be incredibly useful.
